requirements.txt

```diff
@@ -3,6 +3,8 @@ bcrypt==2.0.0
 boto==2.40.0
 cffi==1.8.3
 CherryPy==6.0.1
+functools32==3.2.3.post2
```

Comment:
This file is not actually used in CI or during pip install, the authoritative listing is in setup.py. @jbeezley can better explain the purpose of requirements.txt.